core/primer.md

```diff
@@ -2,6 +2,185 @@
 
 <!-- no verify-specs -->
 
+## Abstract
+
+This non-normative document provides an overview of the xRegistry specification.
+It is meant to complement the xRegistry specification to provide additional
+background and insight into the history and design decisions made during the
+development of the specification. This allows the specification itself to focus
+on the normative technical details.
+
+## Table of Contents
+
+- [History](#history)
+- [Motivation](#motivation)
+- [Design Goals](#design-goals)
+- [Representations](#representations)
+- [Possible Use Cases](#possible-use-cases)
+- [Design decisions or topics of
+  interest](#design-decisions-or-topics-of-interest)
+
+## History
+
+The CNCF Serverless Working group was originally created by the CNCF's Technical
+Oversight Committee to investigate Serverless Technology and to recommend some
+possible next steps for some CNCF related activities in this space. After
+creating the common event format [CloudEvents](https://github.com/cloudevents)
```

Comment:
I wouldn't say "comment event format", while we do have the structured mode, the idea of a CEF (meaning, one event shape to rule the all) has failed many times and that's not really what CE is. It's just about "common metadata, in a well defined location". I'd just leave it as "After creating [CloudEvents]..."